src/hooks/Options.py

```diff
@@ -33,39 +32,21 @@ class TotalCharactersToWinWith(Range):
     range_end = 50
     default = 50
 
-class Goal(Choice): #Don't add this in before_options_defined as "goal" or you will get a warning in the console if you have multiple victory locations
-    """Example to convert"""
-    option_test = 0
-    option_b = 1
-    alias_c = 1
-    default = 1
-
 # This is called before any manual options are defined, in case you want to define your own with a clean slate or let Manual define over them
 def before_options_defined(options: dict) -> dict:
     return options
 
 # This is called after any manual options are defined, in case you want to see what options are defined or want to modify the defined options
-def after_options_defined(options: dict) -> dict:
-    # The generated goal option will not keep your defined values or documentation string unless you add them to option.json or you add them here:
-    # To automatically convert your own Goal class to alias of the generated goal uncomment the lines below and replace 'Goal' with your own option of type Choice
+def after_options_defined(options: PerGameCommonOptions):
+    # To access a modifiable version of options check the dict in options.__annotations__
+    # For example if you want to change DLC_enabled's display name you would do:
+    # options.__annotations__["DLC_enabled"].display_name = "New Display Name"
```

Comment:
This is technically incorrect.  We're passing in a `Type[PerGameCommonOptions]`, not an instance of PerGameCommonOptions.

Also, your code all works, but I'd prefer we use the the [type_hints](https://github.com/ArchipelagoMW/Archipelago/blob/main/Options.py#L1242) property over `__annotations__`.

```suggestion
def after_options_defined(options: Type[PerGameCommonOptions]):
    # To access a modifiable version of options check the dict in options.type_hints
    # For example if you want to change DLC_enabled's display name you would do:
    # options.type_hints["DLC_enabled"].display_name = "New Display Name"
```

Reply:
I used debugging to find a way that works. If this is better, then we use that

Reply:
Oh, your suggestion didn't include the other \__anotation__. I'll do a real commit once I'm home